src/main/java/org/mockbukkit/mockbukkit/entity/PlayerMock.java

```diff
@@ -1973,6 +1973,8 @@ public int getLevel()
 	@Override
 	public void setLevel(int level)
 	{
+		if (level < 0)
```

Comment:
> Note: Paper use Preconditions.checkArgument instead of new IllegalArgumentException. But this copy how current setExp do

We use Preconditions.checkArgument as well, so might as well use that here :)